dts/arm/st/f0/stm32f0.dtsi

```diff
@@ -197,6 +199,8 @@
 			compatible = "st,stm32-timers";
 			reg = <0x40000400 0x400>;
 			clocks = <&rcc STM32_CLOCK_BUS_APB1 0x00000002>;
+			interrupts = <16 0>;
+			interrupt-names = "global";
```

Comment:
On some other drivers (I2C, UART) 'combined' is used instead of global.
Shall it be harmonized accross drivers ? which one should be kept ?

Reply:
I followed CubeMX HAL naming convention. @erwango opinions?

Reply:
Looking to CMSIS files, "global" indeed seems the correct naming for Timers.
But interestingly I2C never get the word "global" in CMSIS files.
They sometime get the word "combined" since the IRQ could be combined with EXT line (but not as systematically as we're using it in dts), which might not make sense in the current implementation as drivers are currently not able to share an IRQ. This word was actually selected as opposed to I2C "Event" + "Error" IRQs that are available on some series, but this could make a slight confusion if one looks HAL too closely, as this is not the same meaning (combined inside the IP as opposed to combined with another IP (EXTI in this case). And then "global" would be more appropriate here to avoid confusion.

So I'd say that:
- "global" is correct here
- uart "combined" IRQs should be converted to "global" (not in current PR though)
- I2C "combined" would also need to be renamed to "global" to minimize confusion, but this is rather a nit-picking improvment